Resources/config/yui.yml

```diff
@@ -377,6 +378,12 @@ system:
                 urlview-ez-template:
                     type: 'template'
                     path: %ez_platformui.public_dir%/templates/fields/view/url.hbt
+                ez-media-view:
+                    requires: ['ez-fieldview', 'mediaview-ez-template']
```

Comment:
you should add the YUI module `node-event-html5` (needed for the `error` event on the video/audio elements)